<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Op. Elementales</string>
     </attribute>
     <widget class="QLabel" name="imagen_ini">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>311</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="histograma">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>260</y>
        <width>311</width>
        <height>101</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Abrir">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir Imagen</string>
      </property>
     </widget>
     <widget class="QLabel" name="imagen_fin">
      <property name="geometry">
       <rect>
        <x>412</x>
        <y>30</y>
        <width>311</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Original</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Alterado</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Restaurar">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Restaurar</string>
      </property>
     </widget>
     <widget class="QSlider" name="Scroll">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>340</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="Opcion">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Sumar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Restar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiplicar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dividir</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>300</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Descargar</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Histograma</string>
     </attribute>
     <widget class="QLabel" name="transHisto_ini">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>291</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="transHisto_fin">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>291</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="abrirTransHis">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>280</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir Imagen</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ecualizar">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>320</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ecualizar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>300</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Descargar</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Comb. Imagenes</string>
     </attribute>
     <widget class="QLabel" name="imagen1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>211</width>
        <height>141</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="imagen2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>211</width>
        <height>141</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="abrir1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="abrir2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QLabel" name="imagen12">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>351</width>
        <height>201</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>250</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Sumar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Restar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiplicar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dividir</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>250</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Descargar</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Filtros y dem√°s</string>
     </attribute>
     <widget class="QLabel" name="imagenfc">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>321</width>
        <height>251</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="imagenfcres">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>321</width>
        <height>251</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Abrirfc">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>280</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Opcion_LOT">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>280</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Perspectiva</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Acristalado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mapeo</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>280</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Perspectivas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>320</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Transformaci√≥n</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>320</y>
        <width>86</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Zoom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Voltear</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rotar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inclinar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Affine</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>360</y>
        <width>86</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Lin. Arbitraria</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Canny</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laplace</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>360</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Filtros</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>320</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Descargar</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
